velox/type/Subfield.h

```diff
@@ -193,7 +208,11 @@ class Subfield {
   };
 
  public:
-  explicit Subfield(const std::string& path);
+  // Separators: the customized separators to tokenize field name.
+  explicit Subfield(
+      const std::string& path,
+      const std::shared_ptr<Separators>& separators =
+          std::make_shared<Separators>());
```

Comment:
Can we add a default `static const std::shared_ptr<Separators>& Separators::default()` and use it here so we don't create a new `shared_ptr` for each subfield construction?

Reply:
Thank you for the suggestion. Updated.